smart-contracts/contracts/interfaces/IRelayPoolHistorical.sol

```diff
@@ -0,0 +1,46 @@
+// The role of this interface is to preserve signature that has been used in the past
```

Comment:
I think it's very confusing to call this "historical". What happens when we change again? We have historical-historical? :D Let's use version names based on dates please for clarity.

Reply:
Ha sure ! The idea was to have a single file where we keep all deprec sigs. Also this avoid having versioning, which is not really needed in our case and is not straightforward. 

The end goal is to have a single unified file with all events that can possibly be triggered so we can index them, so a single file makes sense (vs having multiple versions and merge them after).

It we need contract version then commit height in repo is the best approach imo, as we don't release any contracts for external use.

Reply:
the goal is to be able to do this https://github.com/relayprotocol/relay-vaults/pull/399/commits/53564169aa555f14ee99fdd726c07b61f556cce6

Reply:
still, we may need a better name than "historical". Something that reflects "past and present" :P

Reply:
we can also just have `IRelayPoolEvents` that contains only all events with a notice on the ones that are deprecated